cli/test/sepolia-bsc.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+# This script creates two forks (Bsc and Sepolia) and creates an NTT deployment
+# on both of them.
+# It's safe to run these tests outside of docker, as we create an isolated temporary
+# directory for the tests.
+
+set -euox pipefail
+
+BSC_PORT=8545
+SEPOLIA_PORT=8546
+
+anvil --silent --rpc-url https://bsc-testnet-rpc.publicnode.com -p "$BSC_PORT" &
+pid1=$!
+anvil --silent --rpc-url wss://ethereum-sepolia-rpc.publicnode.com -p "$SEPOLIA_PORT" &
+pid2=$!
+
+# check both processes are running
+if ! kill -0 $pid1 || ! kill -0 $pid2; then
+  echo "Failed to start the servers"
+  exit 1
+fi
+
+# create tmp directory
+dir=$(mktemp -d)
+
+cleanup() {
+  kill $pid1 $pid2
+  rm -rf $dir
+}
+
+trap "cleanup" INT TERM EXIT
+
+# devnet private key
+export ETH_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
is this a test key?

Reply:
yes, this is one of the pre-funded wallets anvil gives you (originally comes from
hardhat)